Rubberduck.Core/UI/ToDoItems/ToDoExplorerViewModel.cs

```diff
@@ -233,18 +236,8 @@ public CommandBase OpenTodoSettings
             }
         }
 
-        private NavigateCommand _navigateCommand;
-        public INavigateCommand NavigateCommand
-        {
-            get
-            {
-                if (_navigateCommand != null)
-                {
-                    return _navigateCommand;
-                }
-                return _navigateCommand = new NavigateCommand(_state.ProjectsProvider);
-            }
-        }
+        private readonly Lazy<NavigateCommand> _navigateCommand;
```

Comment:
Is there a reason to make this instantiate lazily? If it's bound to the view it would be forced to materialize as soon as the control its bound to does, which would be immediately after the ctor exited.

Reply:
I have no idea, I just changed the hand crafted one to one using the `Lazy<T>` type.

Reply:
This might be better just loading it in the ctor then - the instantiation of the window is wiring up all of the event handlers to deal with the wpf bindings,  and I'm not sure what expectations the framework has regarding the immediate availability of bound controls.